examples/platform/silabs/efr32/rs911x/rsi_if.c

```diff
@@ -66,6 +66,8 @@ bool hasNotifiedIPV6 = false;
 bool hasNotifiedIPV4 = false;
 #endif /* CHIP_DEVICE_CONFIG_ENABLE_IPV4 */
 bool hasNotifiedWifiConnectivity = false;
+bool is_commissioned             = false;
+uint32_t ticks                   = WLAN_MIN_RETRY_TIMER_TICKS; // 800 ticks is equals to 1 second.
```

Comment:
comment feels odd: where does the 800 come from?
Should the code use some portTICK_RATE_MS to compute things if we care about seconds instead of ticks?